app/index.html

```diff
@@ -51,21 +58,40 @@
     <script src="bower_components/ngstorage/ngStorage.js"></script>
     <script src="bower_components/ion-datetime-picker/release/ion-datetime-picker.min.js"></script>
     <script src="bower_components/angular-base64/angular-base64.js"></script>
+    <script src="bower_components/jshashes/hashes.js"></script>
```

Comment:
We are currently using "gulp-angular-filesort" to load js files in correct order. But definitely, it is not a lazy loading. I will see if we really need this.